packages/lib/apps/getAppOnboardingRedirectUrl.ts

```diff
@@ -0,0 +1,13 @@
+import { AppOnboardingSteps } from "@calcom/lib/apps/appOnboardingSteps";
+
+import { getAppOnboardingUrl } from "./getAppOnboardingUrl";
+
+export const getAppOnboardingRedirectUrl = (slug: string, teamId?: number, eventTypeId?: number) => {
```

Comment:
I notice we're using this function across every callback function. Could we instead generate this redirect URL and add it to the OAuth state to be read on at the callback endpoint? 

The reason why is when we're developing new apps it's extra mental overhead to remember to add the `getAppOnboardingRedirectUrl` function to every callback endpoint. Whenever we can reduce mental overhead, reduces the possibilities of bugs.